docs/release-notes/2.41.1.md

```diff
@@ -26,6 +26,8 @@ Percona Monitoring and Management (PMM) is an open source database monitoring, m
 
 
 ## Improvements
+- [PMM-12390](https://jira.percona.com/browse/PMM-12390) - [Backup Management] Improved the search functionality for Service Name in MongoDB Backup to be case insensitive.
```

Comment:
```suggestion
- [PMM-12390](https://jira.percona.com/browse/PMM-12390) - [Backup Management]: The **Service name** field on the Create Backup pages is now case-insensitive, allowing for more comprehensive and accurate results retrieval.
```

Reply:
Pls note, I also removed the reference to MongoDB because from what I checked this issue happened before selecting a technology. Let me know if this is not correct and needs to be added back in.